frontend/src/models/dashboardsModel.tsx

```diff
@@ -234,22 +235,40 @@ export const dashboardsModel = kea<dashboardsModelType>({
     },
 
     selectors: ({ selectors }) => ({
+        nameCompareFunction: [
+            () => [() => 1],
```

Comment:
Since this doesn't depend on the logic in any way, let's just pull it out as a function